torch/csrc/jit/runtime/static/native_ops.cpp

```diff
@@ -1511,8 +1511,8 @@ REGISTER_NATIVE_OPERATOR_FUNCTOR(
       return [](ProcessedNode* pnode) {
         const auto& self = pnode->Input(0).toTensor();
         const auto dim = pnode->Input(1).toInt();
-        const auto index = pnode->Input(2).toInt();
-        pnode->Output(0) = at::native::select(self, dim, index);
+        const auto index = pnode->Input(2).toSymInt();
+        pnode->Output(0) = at::native::select_symint(self, dim, index);
```

Comment:
This is a real BC break, add the old overload to the BC header; there are examples

Reply:
Heads up: there's already a call to `tensor_split_sections_symint` in this file. I assume this is also a bug (BC break) and needs to be defined in `ATen/native/NonSymbolicBC.h` as well, right?

Reply:
Well, I guess since `tensor_split_sections_symint` successfully landed and no one complained, I guess it is fine to leave alone. We're using static runtime as a canary for potential other bc breakage in fbcode, which will be a pain for you to fix since you don't have access to that codebase. Anyway, template the thing, put it in BC.h, no problemo.